### ISSUE TITLE
代码中loss的问题

### ISSUE THREAD

**Issue by wangkangdegithub** [sladesha/machine_learning#3]
p = w_0 + dataMatrix[x] * w + interaction
loss = sigmoid(classLabels[x] * p[0, 0]) - 1  
w_0 = w_0 - alpha * loss * classLabels[x]

对于这个loss，我的理解是损失函数sigmod(classLabels[x] * p[0, 0])对classLabels[x] * p[0, 0]求导值，即梯度值，但为什么损失函数是sigmod(y_pred*y_truth)这种形式？
